tests/libcxx/expected_results.txt

```diff
@@ -443,6 +443,9 @@ std/concepts/concepts.compare/concept.equalitycomparable/equality_comparable_wit
 std/containers/sequences/array/array.cons/initialization.pass.cpp:0 FAIL
 std/containers/sequences/array/array.cons/initialization.pass.cpp:1 FAIL
 
+# DevCom-10456523: MSVC incorrectly raises constant evaluation failure in pointer comparison
+std/algorithms/robust_against_adl.compile.pass.cpp:0 FAIL

```

Comment:
It seems that we should mark `:1` `FAIL`, but the case is currently "automatically" `SKIPPED`.

Reply:
It isn't - we've configured PR checks to skip the ASAN configurations, but the ASAN CI will run them.

In general, if `expected_results.txt` has to mention `:0` (plain MSVC), it must also mention `:1` (ASAN), unless the problem is specific to static analysis, as we don't run `/analyze` with ASAN (this is specifically commented whenever it happens).

You can manually see this failure with `-Dtags=ASAN`:

```
D:\GitHub\STL\out\x64>python tests\utils\stl-lit\stl-lit.py -o testing_x64.log -Dtags=ASAN --order=random %STL%\llvm-project\libcxx\test\std\algorithms
-- Testing: 951 tests, 32 workers --
[...]
FAIL: libc++ :: std/algorithms/robust_against_adl.compile.pass.cpp:1 (710 of 951)
[...]
********************
Failed Tests (1):
  libc++ :: std/algorithms/robust_against_adl.compile.pass.cpp:1


Testing Time: 53.42s

Total Discovered Tests: 951
  Skipped          : 644 (67.72%)
  Unsupported      :  22 (2.31%)
  Passed           : 262 (27.55%)
  Expectedly Failed:  22 (2.31%)
  Failed           :   1 (0.11%)

D:\GitHub\STL\out\x64>python %STL%\tools\scripts\print_failures.py testing_x64.log
code: FAIL
name: libc++ :: std/algorithms/robust_against_adl.compile.pass.cpp:1
output: Build setup steps:
Build steps:
Build step failed unexpectedly.
Command: "C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.39.33519\bin\HostX64\x64\cl.EXE" "-c" "D:\GitHub\STL\llvm-project\libcxx\test\std\algorithms\robust_against_adl.compile.pass.cpp" "-ID:\GitHub\STL\out\x64\out\inc" "-ID:\GitHub\STL\llvm-project\libcxx\test\support" "-ID:\GitHub\STL\tests\std\include" "/nologo" "/Od" "/W4" "/w14061" "/w14242" "/w14265" "/w14582" "/w14583" "/w14587" "/w14588" "/w14749" "/w14841" "/w14842" "/w15038" "/w15214" "/w15215" "/w15216" "/w15217" "/w15262" "/sdl" "/WX" "/D_ENABLE_STL_INTERNAL_CHECK" "/bigobj" "/EHsc" "/MTd" "/std:c++latest" "/permissive-" "/utf-8" "/FImsvc_stdlib_force_include.h" "/FIvso1913897.hpp" "/wd4643" "/D_STL_CALL_ABORT_INSTEAD_OF_INVALID_PARAMETER" "-fsanitize=address" "/Zi"
Exit Code: 2 (0x2)
Standard Output:
--
robust_against_adl.compile.pass.cpp
D:\GitHub\STL\llvm-project\libcxx\test\std\algorithms\robust_against_adl.compile.pass.cpp(230): error C2131: expression did not evaluate to a constant
D:\GitHub\STL\out\x64\out\inc\algorithm(429): note: failure was caused by a conversion from void* to a pointer-to-object type
D:\GitHub\STL\llvm-project\libcxx\test\std\algorithms\robust_against_adl.compile.pass.cpp(230): note: the call stack of the evaluation (the oldest call first) is
D:\GitHub\STL\llvm-project\libcxx\test\std\algorithms\robust_against_adl.compile.pass.cpp(230): note: while evaluating function 'bool all_the_algorithms(void)'
D:\GitHub\STL\llvm-project\libcxx\test\std\algorithms\robust_against_adl.compile.pass.cpp(56): note: while evaluating function '_FwdIt **std::adjacent_find<Holder<Incomplete>**,std::equal_to<void *>>(const _FwdIt,_FwdIt,_Pr)'
        with
        [
            _FwdIt=Holder<Incomplete> **,
            _Pr=std::equal_to<void *>
        ]
--


==================================================
```

Reply:
Oh, thanks! I've marked it `FAIL`.